packages/tradershub/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -49,7 +49,7 @@ const JurisdictionModal = () => {
 
     return (
         <DynamicLeverageContext.Provider value={{ isDynamicLeverageVisible, toggleDynamicLeverage }}>
-            <Modal className='bg-background-primary-container'>
+            <Modal className='overflow-y-hidden bg-background-primary-container'>
                 {!isDynamicLeverageVisible ? <Modal.Header title={jurisdictionTitle} /> : null}
                 <Modal.Content>
                     {isDynamicLeverageVisible && <DynamicLeverageTitle />}
```

Comment:
@shontzu-deriv just a suggestion for the future, don't need to change it for now, because I already approved this PR. But I guess you have to follow one code style, or using conditional rendering (`condition && component`) or use ternary operator with null (`condition ? component : null`). Don't need to use both approach in 2 consecutive lines of code)